src/api/PartsService.ts

```diff
@@ -1,8 +1,18 @@
 import axios, { AxiosResponse } from 'axios';
 import { PartsInterface } from '../models/PartsInterface';
 
-export default class PartsService {
-  static async getPartsData(): Promise<AxiosResponse<PartsInterface[]>> {
-    return axios.get<PartsInterface[]>(`${process.env.SERVER}:${process.env.PORT}/parts`);
+class PartsService {
```

Comment:
your service classes accepts the same constructor and fields, it would be better to create common class for them and extend it in services classes

Reply:
done